app/javascript/crayons/CTAs/__stories__/CTAs.mdx

```diff
@@ -27,13 +25,8 @@ By default, the icon itself won't be customizable. Icons within CTAs are hidden
 
 #### Icon-only
 
-There are certain instances where an icon will suffice in place of a text label, but use these cautiously.
-Icon-only CTAs should always use clear and obvious icon, ideally accompanied by a title. Icon-only CTAs will require an `aria-label` to ensure they are understood by users of assistive technologies.
-
-##### Fully rounded
-
-In most cases, whenever we display icon-only CTAs we tend to make them fully rounded - use `rounded` prop for that.
-Other than that please avoid customizing border-radius on CTAs.
+Call To Actions should never come without label, therefore it is forbidden to create CTAs with icon only.
+For such a scenario you should use `<Link />s` or `<Button />s` (depending on use case).
```

Comment:
nitpick (blocking): Call To Actions shouldn't be capitalized for each word. Also suggesting some wording.

```suggestion
A call to action requires a label, therefore it is forbidden to create a CTA with an icon only.
For such a scenario, use a `<Link />` or `<Button />` component (depending on the use case).
```